src/script_chainer/win_exe/script_runner.py

```
@@ -358,7 +358,14 @@ def _cleanup_active_pm():
 
 
 def _on_exit_signal(signum, frame):
-    """控制台关闭/Ctrl+C 时清理子进程并退出。"""
+    """控制台关闭/Ctrl+C 时清理子进程并退出。
+
+    SIGINT/SIGTERM: 通过 sys.exit 触发正常 Python 退出（atexit/finally 可执行）。
```

Ruff (0.15.7)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)